### PR TITLE
Feat: Add pdf to export dataset

### PR DESCRIPTION
**Pull request by arimet**

### CODE REVIEW

File: src/api/controller/api/pdf.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    5: 'Juin',
+    6: 'Juillet',
+    7: 'Août',
+    8: 'Septembre',
+    9: 'Octobre',
+    10: 'Novembre',
+    11: 'Décembre',
+};
+
+function getDateInFrench(date) {
```
Comment (thibault-barrat) [inline src/api/controller/api/pdf.js]: What if the user use english ? Maybe we can get the locale used in front and use moment library to format the date accordingly ?

File: src/api/controller/api/pdf.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+        // Create a document
+        const doc = new PDFDocument();
+
+        // Pipe its output somewhere, like to a file or HTTP response
+        doc.pipe(fs.createWriteStream('/tmp/publication.pdf'));
+
+        doc.font('Helvetica-Bold')
+            .fontSize(25)
+            .text('Données publiées', { align: 'center' });
```
Comment (thibault-barrat) [inline src/api/controller/api/pdf.js]: Smae than date, if we can translate it according to the locale it should be great

**Review by thibault-barrat — CHANGES_REQUESTED** [review]

**Review by thibault-barrat — APPROVED** [review]

**Comment by touv** [comment]
How it happens if there are several thousand documents ? Is there a limit? Is there a way to disable/enable this new option ?

**Comment by arimet** [comment]
We have added a limit that can be modified via the config. 
The display of the option is also done through the config. 

We can imagine in the future an interface to manage this point

**Review by thibault-barrat — APPROVED** [review]

**Pull request merged**